src/middlewared/middlewared/plugins/iscsi_/utils.py

```diff
@@ -1,10 +1,18 @@
+from enum import StrEnum
+
 import bidict
 
 
+class IscsiAuthType(StrEnum):
```

Comment:
Is it a problem that this class name collides with IscsiAuthType in `api/base/types/iscsi.py`?

Reply:
I don't believe this is a problem.  The one in `api/base/types/iscsi.py` is only used for API definitions.

Reply:
Maybe `IscsiAuthTypeEnum`?

Reply:
Okay.